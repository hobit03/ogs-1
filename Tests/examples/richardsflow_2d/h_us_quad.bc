GeoSys-BC: Boundary Conditions ------------------------------------------------
#BOUNDARY_CONDITION
 $PCS_TYPE
  RICHARDS_FLOW
 $PRIMARY_VARIABLE
  PRESSURE1
 $GEO_TYPE
  POLYLINE BOTTOM 
 $DIS_TYPE
  CONSTANT -21500.
#BOUNDARY_CONDITION
 $PCS_TYPE
  RICHARDS_FLOW
 $PRIMARY_VARIABLE
  PRESSURE1
 $GEO_TYPE
  POLYLINE TOP
 $DIS_TYPE
  CONSTANT 0.0
#STOP

#BOUNDARY_CONDITIONS_PRESSURE
0 0 0 200 0 
0 0 0 0 -21500 
#BOUNDARY_CONDITIONS_SATURATION
0 0 0 200 1.00000
0 0 0 0 0.47000

