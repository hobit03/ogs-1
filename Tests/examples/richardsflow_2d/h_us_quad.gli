#POINTS
0 0.00 2 0
1 0.02 2 0
2 0.04 2 0
3 0.00 0 0
4 0.02 0 0
5 0.04 0 0
#POLYLINE
 $NAME 
  TOP
 $POINTS
  0
  1
  2
#POLYLINE
 $NAME 
  BOTTOM
 $POINTS
  3
  4
  5
#POLYLINE
 $NAME 
  OUT
 $POINTS
  0
  1
  2
#STOP

