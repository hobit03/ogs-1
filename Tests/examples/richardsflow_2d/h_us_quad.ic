GeoSys-IC  Initial Conditions ------------------------------------------------
#INITIAL_CONDITION
 $PCS_TYPE
  RICHARDS_FLOW
 $PRIMARY_VARIABLE
  PRESSURE1
 $GEO_TYPE
  DOMAIN 
 $DIS_TYPE
  CONSTANT  -21500
#STOP

; C1.5 Initial Conditions ---------------------------------------------
#INITIAL_CONDITIONS_PRESSURE
 0 0 -21500
;#INITIAL_CONDITIONS_SATURATION
; 0 0 0.530000000000000000000 