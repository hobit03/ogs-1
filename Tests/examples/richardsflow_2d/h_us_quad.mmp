GeoSys-MMP  Material Medium Properties ------------------------------------
#MEDIUM_PROPERTIES
 $GEOMETRY_DIMENSION
  2             //2 für XY, 3für XZ, wenn Gravity auf Z betrachtet.
 $GEOMETRY_AREA
  1.0
 $POROSITY
  1  0.38
 $TORTUOSITY
  1  1.000000e+000
 $PERMEABILITY_TENSOR
  ISOTROPIC  4.46e-13
 $PERMEABILITY_SATURATION
  0	2   
  0	2   
 $CAPILLARY_PRESSURE
  0      1
#STOP


#SOIL_PROPERTIES
 1                    ; element dimension
 1.                   ; element exchange area
 0                    ; porosity model
 .38   1.             ; porosity, tortuosity
 0                    ; mobile immobile model
 0                    ; lithological component
 0                    ; maximum sorption model
; flow
 0                    ; non-linear flow behavior
 .00000               ; storativity
 0   0   4.46e-13     ; permeability model, permeability tensor, permeabilities
1.0  2.0  2.0  1.0  1.0  1.0  1.0 
1.0  1.0  1.0  1.0  1.0  1.0  1.0
; mass transport
 0. 0.                ; long. and trans. dispersion lengths
; heat transport
 0. 0.                ; long. and trans. dispersion lengths
; solid properties
 0. 0.                ; density, heat capacity
 0  0.  
 
 
   4  0.2 0.8 2.