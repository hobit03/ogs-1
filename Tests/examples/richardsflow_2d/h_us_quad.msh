#FEM_MSH
 $PCS_TYPE
  RICHARDS_FLOW
 $NODES
  153
0   0	  0	  0.0     
1   0	  .04	  0.0     
2   .02	  .04	  0.0     
3   .02	  0	  0.0     
4   0	  .08	  0.0     
5   .02	  .08	  0.0     
6   0	  .12	  0.0     
7   .02	  .12	  0.0     
8   0	  .16	  0.0     
9   .02	  .16	  0.0     
10  0	  .2	  0.0     
11  .02	  .2	  0.0     
12  0	  .24	  0.0     
13  .02	  .24	  0.0     
14  0	  .28	  0.0     
15  .02	  .28	  0.0     
16  0	  .32	  0.0     
17  .02	  .32	  0.0     
18  0	  .36	  0.0     
19  .02	  .36	  0.0     
20  0	  .4	  0.0     
21  .02	  .4	  0.0     
22  0	  .44	  0.0     
23  .02	  .44	  0.0     
24  0	  .48	  0.0     
25  .02	  .48	  0.0     
26  0	  .52	  0.0     
27  .02	  .52	  0.0     
28  0	  .56	  0.0     
29  .02	  .56	  0.0     
30  0	  .6	  0.0     
31  .02	  .6	  0.0     
32  0	  .64	  0.0     
33  .02	  .64	  0.0     
34  0	  .68	  0.0     
35  .02	  .68	  0.0     
36  0	  .72	  0.0     
37  .02	  .72	  0.0     
38  0	  .76	  0.0     
39  .02	  .76	  0.0     
40  0	  .8	  0.0     
41  .02	  .8	  0.0     
42  0	  .84	  0.0     
43  .02	  .84	  0.0     
44  0	  .88	  0.0     
45  .02	  .88	  0.0     
46  0	  .92	  0.0     
47  .02	  .92	  0.0     
48  0	  .96	  0.0     
49  .02	  .96	  0.0     
50  0	  1	  0.0     
51  .02	  1	  0.0     
52  0	  1.04	  0.0     
53  .02	  1.04	  0.0     
54  0	  1.08	  0.0     
55  .02	  1.08	  0.0     
56  0	  1.12	  0.0     
57  .02	  1.12	  0.0     
58  0	  1.16	  0.0     
59  .02	  1.16	  0.0     
60  0	  1.2	  0.0     
61  .02	  1.2	  0.0     
62  0	  1.24	  0.0     
63  .02	  1.24	  0.0     
64  0	  1.28	  0.0     
65  .02	  1.28	  0.0     
66  0	  1.32	  0.0     
67  .02	  1.32	  0.0     
68  0	  1.36	  0.0     
69  .02	  1.36	  0.0     
70  0	  1.4	  0.0     
71  .02	  1.4	  0.0     
72  0	  1.44	  0.0     
73  .02	  1.44	  0.0     
74  0	  1.48	  0.0     
75  .02	  1.48	  0.0     
76  0	  1.52	  0.0     
77  .02	  1.52	  0.0     
78  0	  1.56	  0.0     
79  .02	  1.56	  0.0     
80  0	  1.6	  0.0     
81  .02	  1.6	  0.0     
82  0	  1.64	  0.0     
83  .02	  1.64	  0.0     
84  0	  1.68	  0.0     
85  .02	  1.68	  0.0     
86  0	  1.72	  0.0     
87  .02	  1.72	  0.0     
88  0	  1.76	  0.0     
89  .02	  1.76	  0.0     
90  0	  1.8	  0.0     
91  .02	  1.8	  0.0     
92  0	  1.84	  0.0     
93  .02	  1.84	  0.0     
94  0	  1.88	  0.0     
95  .02	  1.88	  0.0     
96  0	  1.92	  0.0     
97  .02	  1.92	  0.0     
98  0	  1.96	  0.0     
99  .02	  1.96	  0.0     
100 0.0	  2	  0.0     
101 .02	  2	  0.0     
102 .04	  .04	  0.0     
103 .04	  0	  0.0     
104 .04	  .08	  0.0     
105 .04	  .12	  0.0     
106 .04	  .16	  0.0     
107 .04	  .2	  0.0     
108 .04	  .24	  0.0     
109 .04	  .28	  0.0     
110 .04	  .32	  0.0     
111 .04	  .36	  0.0     
112 .04	  .4	  0.0     
113 .04	  .44	  0.0     
114 .04	  .48	  0.0     
115 .04	  .52	  0.0     
116 .04	  .56	  0.0     
117 .04	  .6	  0.0     
118 .04	  .64	  0.0     
119 .04	  .68	  0.0     
120 .04	  .72	  0.0     
121 .04	  .76	  0.0     
122 .04	  .8	  0.0     
123 .04	  .84	  0.0     
124 .04	  .88	  0.0     
125 .04	  .92	  0.0     
126 .04	  .96	  0.0     
127 .04	  1	  0.0     
128 .04	  1.04	  0.0     
129 .04	  1.08	  0.0     
130 .04	  1.12	  0.0     
131 .04	  1.16	  0.0     
132 .04	  1.2	  0.0     
133 .04	  1.24	  0.0     
134 .04	  1.28	  0.0     
135 .04	  1.32	  0.0     
136 .04	  1.36	  0.0     
137 .04	  1.4	  0.0     
138 .04	  1.44	  0.0     
139 .04	  1.48	  0.0     
140 .04	  1.52	  0.0     
141 .04	  1.56	  0.0     
142 .04	  1.6	  0.0     
143 .04	  1.64	  0.0     
144 .04	  1.68	  0.0     
145 .04	  1.72	  0.0     
146 .04	  1.76	  0.0     
147 .04	  1.8	  0.0     
148 .04	  1.84	  0.0     
149 .04	  1.88	  0.0     
150 .04	  1.92	  0.0     
151 .04	  1.96	  0.0     
152 .04	  2	  0.0     
$ELEMENTS
 100
0   0  -1   quad 0 3 2 1
1   0  -1   quad 1 2 5 4
2   0  -1   quad 4 5 7 6
3   0  -1   quad 6 7 9 8
4   0  -1   quad 8 9 11 10
5   0  -1   quad 10 11 13 12
6   0  -1   quad 12 13 15 14
7   0  -1   quad 14 15 17 16
8   0  -1   quad 16 17 19 18
9   0  -1   quad 18 19 21 20
10  0  -1   quad 20 21 23 22
11  0  -1   quad 22 23 25 24
12  0  -1   quad 24 25 27 26
13  0  -1   quad 26 27 29 28
14  0  -1   quad 28 29 31 30
15  0  -1   quad 30 31 33 32
16  0  -1   quad 32 33 35 34
17  0  -1   quad 34 35 37 36
18  0  -1   quad 36 37 39 38
19  0  -1   quad 38 39 41 40
20  0  -1   quad 40 41 43 42
21  0  -1   quad 42 43 45 44
22  0  -1   quad 44 45 47 46
23  0  -1   quad 46 47 49 48
24  0  -1   quad 48 49 51 50
25  0  -1   quad 50 51 53 52
26  0  -1   quad 52 53 55 54
27  0  -1   quad 54 55 57 56
28  0  -1   quad 56 57 59 58
29  0  -1   quad 58 59 61 60
30  0  -1   quad 60 61 63 62
31  0  -1   quad 62 63 65 64
32  0  -1   quad 64 65 67 66
33  0  -1   quad 66 67 69 68
34  0  -1   quad 68 69 71 70
35  0  -1   quad 70 71 73 72
36  0  -1   quad 72 73 75 74
37  0  -1   quad 74 75 77 76
38  0  -1   quad 76 77 79 78
39  0  -1   quad 78 79 81 80
40  0  -1   quad 80 81 83 82
41  0  -1   quad 82 83 85 84
42  0  -1   quad 84 85 87 86
43  0  -1   quad 86 87 89 88
44  0  -1   quad 88 89 91 90
45  0  -1   quad 90 91 93 92
46  0  -1   quad 92 93 95 94
47  0  -1   quad 94 95 97 96
48  0  -1   quad 96 97 99 98
49  0  -1   quad 98 99 101 100
50  0  -1   quad 3 103 102 2
51  0  -1   quad 2 102 104 5
52  0  -1   quad 5 104 105 7
53  0  -1   quad 7 105 106 9
54  0  -1   quad 9 106 107 11
55  0  -1   quad 11 107 108 13
56  0  -1   quad 13 108 109 15
57  0  -1   quad 15 109 110 17
58  0  -1   quad 17 110 111 19
59  0  -1   quad 19 111 112 21
60  0  -1   quad 21 112 113 23
61  0  -1   quad 23 113 114 25
62  0  -1   quad 25 114 115 27
63  0  -1   quad 27 115 116 29
64  0  -1   quad 29 116 117 31
65  0  -1   quad 31 117 118 33
66  0  -1   quad 33 118 119 35
67  0  -1   quad 35 119 120 37
68  0  -1   quad 37 120 121 39
69  0  -1   quad 39 121 122 41
70  0  -1   quad 41 122 123 43
71  0  -1   quad 43 123 124 45
72  0  -1   quad 45 124 125 47
73  0  -1   quad 47 125 126 49
74  0  -1   quad 49 126 127 51
75  0  -1   quad 51 127 128 53
76  0  -1   quad 53 128 129 55
77  0  -1   quad 55 129 130 57
78  0  -1   quad 57 130 131 59
79  0  -1   quad 59 131 132 61
80  0  -1   quad 61 132 133 63
81  0  -1   quad 63 133 134 65
82  0  -1   quad 65 134 135 67
83  0  -1   quad 67 135 136 69
84  0  -1   quad 69 136 137 71
85  0  -1   quad 71 137 138 73
86  0  -1   quad 73 138 139 75
87  0  -1   quad 75 139 140 77
88  0  -1   quad 77 140 141 79
89  0  -1   quad 79 141 142 81
90  0  -1   quad 81 142 143 83
91  0  -1   quad 83 143 144 85
92  0  -1   quad 85 144 145 87
93  0  -1   quad 87 145 146 89
94  0  -1   quad 89 146 147 91
95  0  -1   quad 91 147 148 93
96  0  -1   quad 93 148 149 95
97  0  -1   quad 95 149 150 97
98  0  -1   quad 97 150 151 99
99  0  -1   quad 99 151 152 101
#STOP
