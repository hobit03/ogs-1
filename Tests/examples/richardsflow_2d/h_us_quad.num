GeoSys-NUM: Numerical Parameter ----------------------------------------
#NUMERICS
 $PCS_TYPE
  RICHARDS_FLOW
 $ELE_UPWINDING
  0.5
 $ELE_MASS_LUMPING
  1
 $LINEAR_SOLVER 
; method error_tolerance max_iterations theta precond storage
  3      6      1.e-010       1000       1.0    101     4
 $NON_LINEAR_SOLVER 
; method error_tolerance max_iterations relaxation
  PICARD   1.0e-3            10             0.0
#STOP
