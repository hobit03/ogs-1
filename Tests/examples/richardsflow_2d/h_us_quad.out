GeoSys-OUT: Output ------------------------------------------------
#OUTPUT // profile
 $PCS_TYPE
  RICHARDS_FLOW
 $NOD_VALUES
  PRESSURE1
 $GEO_TYPE
  DOMAIN
 $TIM_TYPE
 STEPS 1
 $DAT_TYPE
  PVD
#STOP


